src/util/sss_semanage.c

```diff
@@ -73,44 +73,58 @@ static void sss_semanage_close(semanage_handle_t *handle)
     semanage_handle_destroy(handle);
 }
 
-static semanage_handle_t *sss_semanage_init(void)
+int sss_semanage_init(semanage_handle_t **_handle)
 {
     int ret;
     semanage_handle_t *handle = NULL;
 
     handle = semanage_handle_create();
     if (!handle) {
         DEBUG(SSSDBG_CRIT_FAILURE, "Cannot create SELinux management handle\n");
-        return NULL;
+        ret = EIO;
+        goto done;
     }
 
     semanage_msg_set_callback(handle,
                               sss_semanage_error_callback,
                               NULL);
 
     ret = semanage_is_managed(handle);
-    if (ret != 1) {
-        DEBUG(SSSDBG_CRIT_FAILURE, "SELinux policy not managed\n");
-        goto fail;
+    if (ret == 0) {
+        DEBUG(SSSDBG_TRACE_FUNC, "SELinux policy not managed\n");
```

Comment:
libsemanage documentation says:
```
/* Check whether policy is managed via libsemanage on this system.
 * Must be called prior to trying to connect.
 * Return 1 if policy is managed via libsemanage on this system,
 * 0 if policy is not managed, or -1 on error.
 */
int semanage_is_managed(semanage_handle_t *);
```

So I would use debug message "SELinux policy not managed via libsemanage"